technical-translation-glossary/bitcoin_terminology.csv

```diff
@@ -0,0 +1,56 @@
+en,es,fr
+bitcoin,bitcoin,bitcoin
+blockchain,blockchain,blockchain
+proof of work,prueba de trabajo,preuve de travail
+proof-of-work,prueba de trabajo,preuve de travail
+offchain,offchain,offchain
+onchain,onchain,onchain
+coinbase,coinbase,coinbase
+cold storage,cold storage,cold storage
+hash,hash,hash
+hashed time lock contract,"hashed time lock contract","hashed time lock contract"
+mempool,mempool,mempool
+nonce,número aleatorio,nombre aleatoire
+opcode,opcode,opcode
+halving,halving,halving
+fork,fork,fork
+hard fork,hard fork,hard fork
+soft fork,soft fork,soft fork
+time lock,time lock,time lock
+timelock,timelock,timelock
+locktime,locktime,locktime
+lock time,lock time,lock time
+multisignature,multifirma,multisignature
+orphan block,bloque huérfano,bloc orphelin
+payment channel,canal de pago,canal de paiement
+Script,Script,Script
+script,script,script
+miniscript,miniscript,miniscript
+descriptors,descriptors,descriptors
+output descriptors,output descriptors,output descriptors
+merkle tree,árbol de Merkle,arbre de Merkle
+merkle root,raíz de Merkle,racine de Merkle
+scalability,escalabilidad,scalabilité
+segregated witness,testigo segregado,segregated witness
+double spending,gasto doble,double dépense
+double spend,gasto doble,double dépense
+bolt,BOLT,BOLT
+BOLT,BOLT,BOLT

```

Comment:
does capitalization matter? What about the plural?

Reply:
For AWS, capitalization does matter, as do plurals. Should this terminology doc be a glossary or a ready-to-use for AWS? Thinking on it, I'd rather the former, in which case I'll revert anything to do with caps or plurals and write a script to expand this to make it AWS-ready.

Reply:
I think we could have both, a glossary file and a ready-to-use file for AWS. Also, how about a README in the newly created directory explaining what this is about and how it is used?